From e8af28a0cdfd8d10fe2d8742e57217322befb2d9 Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Tue, 14 Nov 2017 19:35:19 -0500
Subject: [PATCH 099/102] Documentation: dt: net: Add TI-PRUeth PRP PTP
 bindings

Add PRP PTP tx interrupt DT binding information for TI's
PRUSS Ethernet device driver.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
---
 Documentation/devicetree/bindings/net/ti-prueth.txt | 16 +++++++++-------
 1 file changed, 9 insertions(+), 7 deletions(-)

diff --git a/Documentation/devicetree/bindings/net/ti-prueth.txt b/Documentation/devicetree/bindings/net/ti-prueth.txt
index 5529b80..7a4e637 100644
--- a/Documentation/devicetree/bindings/net/ti-prueth.txt
+++ b/Documentation/devicetree/bindings/net/ti-prueth.txt
@@ -16,10 +16,12 @@ Must contain children, one for each of the MAC ports.
 Required properties for children:
 - phy-handle       : See ethernet.txt file in the same directory.
 - phy-mode         : See ethernet.txt file in the same directory.
-- interrupt-names  : should be "rx" and "tx"
+- interrupt-names  : should be "rx", "tx" and "ptp_tx".
+                     ptp_tx is optional.
 - interrupts       : should contain an array of PRUSS system event
-                     numbers used as the interrupt sources for Rx
-                     and Tx respectively.
+                     numbers used as the interrupt sources for Rx,
+                     Tx and PTP Tx event packet timestamping (with
+                     PRP firmware) respectively.
 
 Optional properties for children:
 - local-mac-address	: mac address for the port.
@@ -35,8 +37,8 @@ Example (am572x-idk board):
 		pruss2_emac0: ethernet-mii0 {
 			phy-handle = <&pruss2_eth0_phy>;
 			phy-mode = "mii";
-			interrupts = <20>, <22>;
-			interrupt-names = "rx", "tx";
+			interrupts = <20>, <22>, <23>;
+			interrupt-names = "rx", "tx", "ptp_tx";
 			/* Filled in by bootloader */
 			local-mac-address = [00 00 00 00 00 00];
 		};
@@ -44,8 +46,8 @@ Example (am572x-idk board):
 		pruss2_emac1: ethernet-mii1 {
 			phy-handle = <&pruss2_eth1_phy>;
 			phy-mode = "mii";
-			interrupts = <21>, <23>;
-			interrupt-names = "rx", "tx";
+			interrupts = <21>, <23>, <24>;
+			interrupt-names = "rx", "tx", "ptp_tx";
 			/* Filled in by bootloader */
 			local-mac-address = [00 00 00 00 00 00];
 		};
-- 
1.9.1

